### DISCUSSION TITLE
Trying to understand ratio

### DISCUSSION THREAD

**Discussion by baitinghollow** [blakeblackshear/frigate#15716] [General]
Greetings, My understanding is that a ratio of 1 is a bounding box that is square. And anything less than 1 would be  taller and anything more than 1 would be wider. Yet in looking at many boxes, it seems that a square it .50. I have a image of a car that the box is 1.01 and yet the box is twice as long as it is high.  My detect resolution is 480x640 and I'm seeing the boxes in the new Explore tab.  What am I missing
Thanks

**Comment by NickM-27** [comment]
What you're describing is correct, I'll try to confirm if the ratio is being calculated incorrectly.

**Comment by hawkeye217** [comment]
This was actually a small bug in the frontend code, the ratio in the object lifecycle pane wasn't being re-normalized back to the dimensions of the camera frame. Without this adjustment, the result was misleading because it didn't account for the difference in scale between width and height in the original image.

Fixed in https://github.com/blakeblackshear/frigate/pull/15717
